### PR TITLE
Implemented saving hotel registration details to the database

### PR DESCRIPTION
**Pull request by Lakshini**

### CODE REVIEW

File: src/Services/Domain.Services/HotelService.js [hunk 1]
```diff
@@ -19,53 +16,49 @@ class HotelService {
...
+			// Saving to the image table
+			if (data.ImageUrls && data.ImageUrls.length > 0) {
+				for (const url of data.ImageUrls) {
+					const imageEntity = {
+						HotelId: insertedId,
+						Url: url,
+					};
+	
+					if (await this.#dbContext.isTableExists(Tables.IMAGES)) {
+						try {
```
Comment (DinooliSUduwarage) [inline src/Services/Domain.Services/HotelService.js:44]: Should this table be HotelImages than Images only, I think I renamed it

File: src/startup.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const version = require("./../package.json").version;
 
 const voyageLankaContract = async ctx => {
```
Comment (DinooliSUduwarage) [inline src/startup.js:9]: Should be tripQ contract somethin

File: src/Services/Domain.Services/HotelService.js (hunk 1 above)
Comment (Lakshini) [reply src/Services/Domain.Services/HotelService.js:44]: Done

File: src/startup.js (hunk 2 above)
Comment (Lakshini) [reply src/startup.js]: Done

**Review by DinooliSUduwarage — APPROVED** [review]

**Pull request merged**